DonjinKrawler-Client/src/main/java/donjinkrawler/items/Armory.java

```diff
@@ -21,6 +22,9 @@ public Armory clone() throws CloneNotSupportedException {
         return armory;
     }
 
+    @Override
+    public abstract void accept(ItemVisitor v);
```

Comment:
From the [checklist](https://dzone.com/articles/java-code-review-checklist):
`Use intention revealing names.`